### PR TITLE
Mailmodo source doc

### PR DESCRIPTION
**Pull request by ishachopra0401**
## What do these changes do?

Create Mailmodo docs

### CODE REVIEW

**Review by rohithkaza — APPROVED** [review]

File: docs/sources/event-streams/cloud-apps/mailmodo.mdx [hunk 1]
```diff
@@ -0,0 +1,189 @@
+---
+title: "Mailmodo"
+description: Step-by-step guide on ingesting your event data from Mailmodo into RudderStack.
+aliases: ["Mailmodo source"]
+---
+
+[Mailmodo](https://www.mailmodo.com/) is an interactive email marketing tool that provides several features such as sending transactional campaigns, email automation and drip emails with customer journeys, WYSIWYG email editor, and many other tool integrations.
```
Comment (ameypv-rudder) [inline docs/sources/event-streams/cloud-apps/mailmodo.mdx:7]:
```suggestion
[Mailmodo](https://www.mailmodo.com/) is an interactive email marketing tool. It provides several features such as transactional campaigns, email automation and drip emails with customer journeys, WYSIWYG email editor, and many other tool integrations.
```

Comment (ameypv-rudder) [reply docs/sources/event-streams/cloud-apps/mailmodo.mdx:7]: For better readability, as suggested by Grammarly.

File: docs/sources/event-streams/cloud-apps/mailmodo.mdx [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+
+This guide will help you set up Mailmodo as a source in RudderStack.
+
+## Getting started
+
+Follow these steps to set up your Mailmodo source in the RudderStack dashboard:
+
+1. Go to your [RudderStack dashboard](https://app.rudderstack.com/) and click on **Add Source**. From the list of **Event Streams** sources, select **Mailmodo**.
```
Comment (ameypv-rudder) [inline docs/sources/event-streams/cloud-apps/mailmodo.mdx:15]:
```suggestion
1. Go to your [RudderStack dashboard](https://app.rudderstack.com/) and click **Add Source**. From the list of **Event Streams** sources, select **Mailmodo**.
```

File: docs/sources/event-streams/cloud-apps/mailmodo.mdx [hunk 3]
```diff
@@ -0,0 +1,189 @@
...
+2. Assign a name to your source and click **Continue**.
+3. Your Mailmodo source is now configured. Note the **Webhook URL** in the **Settings** tab, as shown:
+
+<img src="../../../assets/event-stream-sources/Mailmodo-2.png" alt="Mailmodo source webhook URL" />
+
+4. Log into your <a href="https://manage.mailmodo.com/auth/logi">Mailmodo account</a> and enter the **Webhook URL** based on the required format. Mailmodo can send data to a webhook in any of the following formats:
```
Comment (ameypv-rudder) [inline docs/sources/event-streams/cloud-apps/mailmodo.mdx:21]:
```suggestion
4. Log into your <a href="https://manage.mailmodo.com/auth/logi">Mailmodo account</a> and enter the **Webhook URL** in the required format. Mailmodo can send data to a webhook in any of the following formats:    
```

Comment (ameypv-rudder) [reply docs/sources/event-streams/cloud-apps/mailmodo.mdx:21]: "Mailmodo can send data to a webhook in any of the following formats: 

- Send data via webhook by setting it up as an action in the user journey..."

Reading it together feels a bit odd. can we say:

 "Mailmodo can send data to a webhook in any of the following ways: 

- By setting it up as an action in the user journey..
- By adding Mailmodo's webhook integration..."

Thoughts?

File: docs/sources/event-streams/cloud-apps/mailmodo.mdx [hunk 4]
```diff
@@ -0,0 +1,189 @@
...
+    "triggerData.eventProperty.Price": "829",
+    "triggerData.journeyId": "3f135bf7-****-4e31-b265-f61cfe1bd423",
+    "triggerData.triggerSource": "New Custom Event Trigger - Product Viewed"
+  },
+  "type": "track"
+}
+```
+
+
+- If you choose to export Mailmodo template submissions to your software stack using Webhook, a sample payload sent by Mailmodo is shown below:
```
Comment (ameypv-rudder) [inline docs/sources/event-streams/cloud-apps/mailmodo.mdx:114]:
```suggestion
- If you choose to export the Mailmodo template submissions to your software stack using a webhook, a sample payload sent by Mailmodo is shown below:
```

File: docs/sources/event-streams/cloud-apps/mailmodo.mdx [hunk 5]
```diff
@@ -0,0 +1,189 @@
...
+
+<img src="../../../assets/event-stream-sources/webhook_url_2.png" alt="Mailmodo source webhook URL" />
+
+## Event transformation
+
+RudderStack ingests the Mailmodo events after converting them into the RudderStack event format. It also maps the following properties from the Mailmodo event payload to the RudderStack properties:
+
+| Mailmodo property  | RudderStack property| Notes |
+| :------------------ | :------------- |:------------- |
+| `mailmodoUserId`  | `context.externalId[0].type`    | Only when [webhook is set up as an action in the user journey](https://support.mailmodo.com/support/solutions/articles/84000365396-send-webhook-as-an-action-in-the-journey). |
```
Comment (ameypv-rudder) [inline docs/sources/event-streams/cloud-apps/mailmodo.mdx:37]:
```suggestion
| `mailmodoUserId`  | `context.externalId[0].type`    | Only when [the webhook is set up as an action in the user journey](https://support.mailmodo.com/support/solutions/articles/84000365396-send-webhook-as-an-action-in-the-journey). |
```

File: docs/sources/event-streams/cloud-apps/mailmodo.mdx [hunk 6]
```diff
@@ -0,0 +1,189 @@
...
+| `triggerData.userId`        | `context.externalId[0].value`        | Only when [webhook is set up as an action in the user journey](https://support.mailmodo.com/support/solutions/articles/84000365396-send-webhook-as-an-action-in-the-journey). |
```
Comment (ameypv-rudder) [inline docs/sources/event-streams/cloud-apps/mailmodo.mdx:38]:
```suggestion
| `triggerData.userId`        | `context.externalId[0].value`        | Only when [the webhook is set up as an action in the user journey](https://support.mailmodo.com/support/solutions/articles/84000365396-send-webhook-as-an-action-in-the-journey). |
```

**Review by ameypv-rudder — APPROVED** [review]

**Pull request merged**